testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts

```diff
@@ -0,0 +1,33 @@
+import { createWidget } from "../../../../helpers/testHelper";
+
+export const widgetHeight = 833;
```

Comment:
`const` should not be separated from the initialization of Scheduler, 'cause of single-use of it